modules/wsl-distro.nix

```diff
@@ -119,15 +119,17 @@ in
       where = "/tmp/.X11-unix/X0";
       type = "none";
       options = "bind";
-      wantedBy = [ "local-fs.target" ];
       after = [ "nixos-wsl-migration-x11mount.service" ];
       wants = after;
+      wantedBy = [ "local-fs.target" ];
     }];
     # Remove symbolic link for WSLg X11 socket, which was created by NixOS-WSL until 2024-02-24
     systemd.services.nixos-wsl-migration-x11mount = {
       description = "Remove /tmp/.X11-unix symlink if present";
-      before = [ "tmp-.X11\x2dunix-X0.mount" ];
```

Comment:
Why did we have this in the first place? if /tmp is a tmpfs we don't need this 🤔

Reply:
Because it's not and we had a hack for this before